src/psyclone/tests/gocean1p0_config_test.py

```diff
@@ -220,8 +260,8 @@ def test_valid_config_files():
     psy, _ = get_invoke("new_iteration_space.f90", "gocean1.0", idx=0)
 
     gen = str(psy.gen)
-    # "# nopep8" suppresses the pep8 warning about trailing white space at end of
-    # line (after the "END DO ")
+    # "# nopep8" suppresses the pep8 warning about trailing white space
```

Comment:
We don't need this any more - I think the trailing whitespace issue is fixed?

Reply:
Indeed - I forgot all about nopep8 :)